UnityProject/Assets/Scripts/Items/Medical/Genetics/Amber.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Amber : MonoBehaviour, ICheckedInteractable<PositionalHandApply>
+{
+	public Stackable Stacking;
+
+	public void Start()
+	{
+		Stacking = this.GetComponent<Stackable>();
+	}
+
+	public bool WillInteract(PositionalHandApply interaction, NetworkSide side)
+	{
+		if (DefaultWillInteract.Default(interaction, side) == false) return false;
+		if (interaction.TargetObject == gameObject) return false;
+		if ( Validations.HasComponent<DNAConsole>(interaction.TargetObject) == false) return false;
```

Comment:
return `Validations.HasComponent<DNAConsole>(interaction.TargetObject)`

Reply:
nah,I feel like that's needed makes it less clear